stdlib/typing.pyi

```diff
@@ -352,7 +352,7 @@ def type_check_only(func_or_cls: _F) -> _F: ...
 
 class _Alias:
     # Class for defining generic aliases for library types.
-    def __getitem__(self, typeargs: Any) -> Any: ...
+    def __getitem__(self, typeargs: Any) -> object: ...
```

Comment:
Hmm, what's the rationale behind this change? It seems somewhat unrelated to the other changes in this PR

Reply:
oops, I think I accidentally unstashed something.